contracts/TokenHolderRevenueFund.sol

```diff
@@ -53,9 +63,11 @@ contract TokenHolderRevenueFund is Ownable, AccrualBeneficiary, Servable, Transf
 
     TxHistoryLib.TxHistory private txHistory;
 
-    mapping(address => mapping(address => mapping(uint256 => uint256[]))) public claimedAccrualBlockNumbersByWalletCurrency;
+    mapping(address => mapping(uint256 => AccrualPeriod[])) public accrualPeriodsByCurrency;
```

Comment:
Maybe consider name change of map as well?

Reply:
Indeed. Now the two fields you mention I have renamed as follows:
```
    mapping(address => mapping(address => mapping(uint256 => uint256[]))) public claimedAccrualIndicesByWalletCurrency;
    mapping(address => mapping(address => mapping(uint256 => mapping(uint256 => bool)))) public accrualClaimedByWalletCurrencyIndex;
```

And the one before that was renamed from `accrualPeriodsByCurrency` to `closedAccrualPeriodsByCurrency`.